oak/server/rust/oak_runtime/src/metrics/mod.rs

```diff
@@ -103,18 +123,95 @@ impl GrpcServerMetrics {
                 &["method_name"],
                 "Histogram of response latency of RPCs handled by the server.",
             )),
-            grpc_response_size_bytes: builder.register(histogram_vec(
-                "grpc_response_size_bytes",
+            grpc_server_response_size_bytes: builder.register(histogram_vec(
+                "grpc_server_response_size_bytes",
                 &["method_name"],
-                "Histogram of response sizes of RPCs handled by the server.",
+                "Histogram of RPC response message sizes sent by the server.",
             )),
             grpc_server_msg_sent_total: builder.register(histogram_vec(
                 "grpc_server_msg_sent_total",
                 &["method_name"],
-                "Total number of RPC response messages sent by the server.",
+                "Histogram of number of RPC response messages sent by the server.",
+            )),
+        }
+    }
+}
+
+impl GrpcClientMetrics {
+    fn new(builder: &MetricsBuilder) -> Self {
+        GrpcClientMetrics {
+            grpc_client_started_total: builder.register(counter_vec(
+                "grpc_client_started_total",
+                &["method_name"],
```

Comment:
One to think about (and defer to a later PR if needed): should all of these labels be `["server", "method_name"]` ?  If an Oak Application connects to multiple external gRPC services, is it going to be useful to group metrics for different services, and would just the `method_name` be fully-qualified enough to allow us to?

(I was sure I'd made this comment before, but GitHub has no record of it &hellip;)